lib/surface/compiler.ex

```diff
@@ -317,6 +318,39 @@ defmodule Surface.Compiler do
      }}
   end
 
+  defp convert_node_to_ast(
+         :unless,
+         {_, attributes, children, node_meta},
+         compile_meta
+       ) do
+    meta = Helpers.to_meta(node_meta, compile_meta)
+
+    default = %AST.AttributeExpr{value: false, original: "", meta: node_meta}
+
+    condition = attribute_value_as_ast(attributes, "condition", default, compile_meta)
+
+    inverted_condition =
+      case condition do
+        %AST.AttributeExpr{value: value} = expr ->
+          value =
+            quote generated: true do
+              !unquote(value)
+            end
+
+          %AST.AttributeExpr{expr | value: value}
+
+        condition ->
+          condition
+      end
+
+    {:ok,
+     %AST.If{
+       condition: inverted_condition,
+       children: to_ast(children, compile_meta),
+       meta: meta
+     }}
```

Comment:
@miguel-s Another way to achieve what you want is to use the new `else` attribute of the `AST.If` struct.

```suggestion
    {:ok,
     %AST.If{
       condition: condition,
       children: [],
       else: to_ast(children, compile_meta),
       meta: meta
     }}
```

Reply:
This makes a lot of sense and would make it trivial to support `#unless..#else` if we ever want to.

Reply:
Oh dear lord! My head just short-circuited now picturing `#unless..#else` in my mind. @miguel-s you're crazy, mate!

Reply:
`unless..else` is a thing in Elixir 😂 https://hexdocs.pm/elixir/Kernel.html#unless/2

I tried using it once...